wavedump/src/fit-histograms

```diff
@@ -0,0 +1,566 @@
+#!/usr/bin/env python3
+from __future__ import print_function, division
+import h5py
+import numpy as np
+import sys
+import csv
+import os
+import ROOT
+from ROOT import gROOT
+from ROOT import TMath
+
+# ROOT.ROOT.EnableThreadSafety()
+
+# DEFAULT VALUES FOR SPE FIT:
+D_OFFSET = 0
+D_LAMBDA = 1
+D_SPE_CHARGE = 0.8
+D_NOISE_SPREAD = 0.01
+D_SPE_CHARGE_SPREAD = 0
+D_ZERO_PEAK_SPREAD = 0.4
+# Number of peaks we use for SPE fitting
+n = 20
```

Comment:
one way to potentially speed this up is to set this dynamically based on your best guess of lambda. If for example, you think the mean number of photons is lambda you can do:

```python
n = max(20,min(2,poisson.ppf(0.95,lambda)))
```

Reply:
I increased the lower bound to 4 since it had a noticeable increase in the accuracy of the estimate, but otherwise I got that implemented now.